src/app/zap-templates/zcl/data-model/chip/temperature-control-cluster.xml

```diff
@@ -35,8 +29,8 @@ limitations under the License.
     <attribute side="server" code="0x0001" define="MIN_TEMP" type="INT16S" writable="false" optional="true">MinTemperature</attribute>
     <attribute side="server" code="0x0002" define="MAX_TEMP" type="INT16S" writable="false" optional="true">MaxTemperature</attribute>
     <attribute side="server" code="0x0003" define="STEP" type="INT16S" writable="false" optional="true">Step</attribute>
```

Comment:
Pre-existing, but why are these using "INT16S" instead of "temperature" for types?